app/assets/stylesheets/govuk_publishing_components/components/_button.scss

```diff
@@ -110,7 +110,7 @@ $gem-quiet-button-hover-colour: darken($gem-quiet-button-colour, 5%);
 
 // Begin adjustments for font baseline offset
 // These should be removed when the font is updated with the correct baseline
-// For the 1px addition please see https://github.com/alphagov/govuk-frontend/pull/365#discussion_r154349428
+// For the 1px addition please see https://github.com/alphagov/govuk/pull/365#discussion_r154349428
```

Comment:
Not sure this one should have been changed from govuk-frontend to govuk, possibly a victim of find and replace?

Reply:
The initial replace was 'erroneous', but the adjustments (and the comments) were removed in a following commit.